drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1808,7 +1808,8 @@ function DragAndDropBlock(runtime, element, configuration) {
             state.attempts = data.attempts;
             state.grade = data.grade;
             state.feedback = data.feedback;
-            state.overall_feedback = data.overall_feedback;
+            const filtered = data.overall_feedback.filter(el => el.message_class !== null)
+            state.overall_feedback = filtered;
```

Comment:
what is the purpose of using additional `const` instead of adding filter function directly on `state.overall_feedback`?
```
state.overall_feedback = data.overall_feedback.filter(el => el.message_class !== null)
```

Reply:
Updated according to comment